packages/webapp/pages/onboarding.tsx

```diff
@@ -158,12 +158,13 @@ export function OnboardPage(): ReactElement {
 
     return router.replace({
       pathname: '/',
-      ...(!onboardingOptimizations && {
-        query: {
+      query: {
+        ua: 'true',
```

Comment:
Based on the above comment, just as with usage of `onboardingOptimizations`, we can use useFeature here for the acquisition and only set the query param if we are on the right version.